src/framework/response/apifail.rs

```diff
@@ -37,6 +38,7 @@ impl PartialEq for ApiError {
 
 impl Eq for ApiError {}
 impl Eq for ApiErrors {}
+impl Error for ApiError {}
```

Comment:
Not ApiErrors?

Reply:
No, ApiErrors doesn't impl Display, which is required for impling Error.